bundles/org.openhab.binding.samsungtv/src/main/java/org/openhab/binding/samsungtv/internal/protocol/RemoteControllerWebSocket.java

```diff
@@ -55,37 +64,33 @@ public class RemoteControllerWebSocket extends RemoteController implements Liste
     private final WebSocketArt webSocketArt;
     private final WebSocketV2 webSocketV2;
 
+    // refresh limit for current app update (in seconds)
+    private static final long UPDATE_CURRENT_APP_REFRESH = 10;
+    private Instant previousUpdateCurrentApp = Instant.ofEpochMilli(0);
```

Comment:
Simplification possible:
```suggestion
    private Instant previousUpdateCurrentApp = Instant.MIN;
```

Reply:
Ok.